pyfr/integrators/dual/pseudo/pseudocontrollers.py

```diff
@@ -136,12 +141,29 @@ def __init__(self, *args, **kwargs):
                     self.backend.kernel(
                         'localerrest', tplargs=tplargs,
                         dims=[ele.nupts, ele.neles], err=err,
-                        errprev=err_prev, dtau_upts=dtaumat
+                        errprev=err_prev, dtau_upts=dtaumat,
                     )
                 )
 
+        self.pseudo_step_multiplied(1.0)
+
         self.backend.commit()
 
+    def adjust_pseudo_step(self, dt):
+        old_dtau = self.dtau
+        super().adjust_pseudo_step(dt)
+        self.pseudo_step_multiplied(self.dtau / old_dtau)
+
+    def pseudo_step_multiplied(self, y):
+        self.dtau_min *= y
+        self.dtau_max *= y
+
+        for k, idx in self.pintgkernels:
+            if k == 'localerrest':
+                for kk in self.pintgkernels[k, idx]:
+                    kk.bind(dtau_min=self.dtau_min, 
+                        dtau_max=self.dtau_max, dtau_fieldf=y)
```

Comment:
Indentation.

Reply:
You mean like this? I tried copying the format in lines 141-144.